airbyte-integrations/connector-templates/source-singer/source_{{snakeCase name}}_singer/source.py.hbs

```diff
@@ -22,20 +22,46 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 """
 
+import json
```

Comment:
big fan of this file. solves this handily: https://github.com/airbytehq/airbyte/issues/816